config/datasetIngest.py

```diff
@@ -6,9 +6,10 @@
 
 decamConfigDir = os.path.join(getPackageDir('obs_decam'), 'config')
 
+config.textDefectPath = os.path.join(getPackageDir('obs_decam_data'), 'decam', 'defects')
 config.dataIngester.retarget(DecamIngestTask)
 config.dataIngester.load(os.path.join(decamConfigDir, 'ingest.py'))
 config.calibIngester.load(os.path.join(decamConfigDir, 'ingestCalibs.py'))
-config.defectIngester.load(os.path.join(decamConfigDir, 'ingestCalibs.py'))
+config.defectIngester.load(os.path.join(decamConfigDir, 'ingestDefects.py'))
 config.defectIngester.parse.extnames = []
```

Comment:
Given that we're now using a dedicated task (and dedicated config file) instead of `IngestCalibsTask`, is this last line still appropriate?

Reply:
You're saying that it should just be picked up from the repo?  If that's the case, why do we need the line above?

Reply:
I'm saying that this line was originally here because the config was piggybacking off the config file for `IngestCalibsTask`, which didn't quite work for defects. Maybe we don't need this because it's the default for defects (or because they're no longer FITS?), or maybe it belongs in `ingestDefects.py`. I'm not familiar enough with the new ingestion code to know.

Reply:
On second thought, I'm pretty sure this line is now redundant. It was there to override [these lines](https://github.com/lsst/obs_decam/blob/master/config/ingestCalibs.py#L5-L10) from `ingestCalibs.py`.

Reply:
I'm happy to remove.  Are the unit tests sufficient to indicate which is correct?

Reply:
I think so? I assume that if the defects were stored in extensions and we omitted `extnames`, they wouldn't get ingested at all.

Reply:
`obs_decam` tests pass, but I'm not sure that ever exercises the `DatasetIngestTask`.